lib/parser/lexer/explanation.rb

```diff
@@ -40,12 +40,12 @@ def decorate(range, token, info)
       from, to = range.begin.column, range.end.column
 
       line = range.source_line + '   '
-      line[from...to] = "\e[4m#{line[from...to]}\e[0m"
+      line[from...to] = Color.underline(line[from...to])
 
       tail_len   = to - from - 1
       tail       = '~' * (tail_len >= 0 ? tail_len : 0)
-      decoration =  "#{" " * from}\e[1;31m^#{tail}\e[0m #{token} ".
-                        ljust(70) + info
+      decoration =  "#{" " * from}#{Color.red("^#{tail}", bold: true)} #{token} ".
+                        ljust(68) + info
```

Comment:
L24 contained unnecessary `0;`, and this change removes the `0;`. So we need to reduce the ljust value to align them.